### PR TITLE
Embed life is a better life?

### PR DESCRIPTION
**Pull request by TBestLittleHelper**
This just feels like a better idea. We could have a " hide embed " button, but not sure it's needed? 

image

### CODE REVIEW

File: src/components/Round.vue [hunk 1]
```diff
@@ -174,9 +172,13 @@ getRound();
     </div>
 
     <h3 class="text-white text-xl my-4">
-      Spectator View
+      Broadcast Round Embed
```
Comment (ornicar) [inline src/components/Round.vue:175]: I don't think the title is necessary. Also the embed should probably not be inside an `h3` element.

**Comment by TBestLittleHelper** [comment]
One arguble downside, is that Stockfish eval is readily availible with the embed. Since the software is quite likely run inside the playing area's we would ideally not have the Stockfish eval shown 🤔  .... I guess use of delay would mitigat this a bit, but maybe we ought to have that " hide embed " button.  Or a param to hide eval in the embed itself ( " Computer analysis disabled 
(click to) Enable ") , would probably be fine.

**Comment by ornicar** [comment]
https://github.com/lichess-org/lila/commit/6e8a08b3998aed789b05f3f375706ba4821967de

File: src/components/Round.vue [hunk 2]
```diff
@@ -173,41 +171,9 @@ getRound();
...
-      <RoundBoard
-        v-for="(game, index) in round.games"
-        v-model="round.games[index]"
-        :showBoard="showBoard"
-        :roundURL="round.round.url"
-        :key="game.id"
-      ></RoundBoard>
-    </div>
+    <iframe
+      :src="'https://lichess.org/embed/broadcast/_/_/' + round.round.id + '?evals=0'"
```
Comment (fitztrev) [inline src/components/Round.vue:175]: `settings.lichessUrl` will use the configurable base url from the Settings page and then it should work with dev envs

**Pull request merged**